src/api-server.ts

```diff
@@ -44,11 +45,10 @@ export class ApiServer {
                 if (err) {
                     return reject(err);
                 }
-
+                const address = this.server.address() as AddressInfo;
                 // TODO: replace with Morgan call
                 // tslint:disable-next-line:no-console
-                console.log(`Listening to http://${this.server.address().address}:${this.server.address().port}`);
-
+                console.log(`Listening to http://localhost:${address.port}`);
```

Comment:
I think that would be better to print 'localhost' only if the address is undefined.

```typescript
console.log(`Listening to http://${address.address? address.address:'localhost'}:${address.port}`);
```